recipes/lightgbm/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "lightgbm" %}
+{% set version = "2.0.4" %}
+{% set sha256 = "6472ded5b0df03504b9d53f4d0e8070a59c2ea015ae526137305c75eb2a77eec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - cmake_osx.patch  # [osx]
```

Comment:
For reference, here is the issue ( https://github.com/Microsoft/LightGBM/issues/710 ). Might be worth adding a link above the patch in the recipe as a comment.